bedrock/firefox/redirects.py

```diff
@@ -520,4 +520,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 1299947
     redirect('^firefox/channel/?$', 'firefox.channel.desktop'),
+
+    # Bug 1277196
+    redirect('^firefox/firstrun/learnmore/$', 'firefox.features'),
```

Comment:
In addition to this redirect, we also need to handle the same URL with a version number e.g. https://www.mozilla.org/en-US/firefox/50.0/firstrun/learnmore/

It's probably easiest to add another redirect for this, using a regexp for the version number:

```
redirect(r'^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/firstrun/learnmore/?$', 'firefox.features'),
```

A test to accompany this extra test would be good too.

```
url_test('/firefox/{49.0,49.0.1,50.0a1,51.0a2}/firstrun/learnmore', '/firefox/features/'),
```